src/main/anovos/data_report/report_preprocessing.py

```diff
@@ -32,9 +32,11 @@ def save_stats(spark, idf, master_path, function_name, reread=False, run_type="l
     :run_type: local or emr based on the mode of execution. Default value is kept as local
     :reread: option to reread. Default value is kept as False
     """
-
+    
     if run_type == "local":
         local_path = master_path
+    elif run_type == "databricks":
+        local_path = "/dbfs/FileStore/tables/report_stats"
     else:
```

Comment:
@ranjanravish As above, I'd explicitly check for all types using `if`/`elif` and raise a `ValueError` if the user gives a different one. Otherwise, there'll be lots of silent failures that are hard to track down.

Reply:
Here also, Added condition that will raise a ValueError to catch typos